microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -87,9 +91,9 @@ The `CoreWebView2Controller` properties and methods:
 
 
 <!-- ====================================================================== -->
-## Windowed hosting
+## Windowed and Window to Visual hosting
 
-Windowed hosting can be described as an HWND that stores information.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  Some of the Output/Input commands are handled for you by the framework you choose; however, you will still need to handle window management.
+In this hosting mode, WebView2 content is hosted directly in a window.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  The benefit of this is taht some of the Input/Output commands are handled for you by the OS or the framework. However, you will still need to handle some window management functionalities. Unless specified, all of the information in this section is also true for Window to Visual hosting mode.
```

Comment:
```suggestion
In the Windowed hosting mode and the "Window to Visual" hosting mode, WebView2 content is hosted directly in a window.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  The benefit of this is that some of the Input/Output commands are handled for you by the OS or by the framework. However, you will still need to handle some window management functionality. 

The information in this section applies to both the Window hosting mode and the "Window to Visual" hosting mode, unless otherwise specified.
```
if you don't get to this change, i'll push a commit thats a writer/editor pass eg typofix

Reply:
So it's actually untrue that in Window to Visual hosting mode, the content is hosted directly in a window. But that is unclear from my 'all information in this section is also true...' statement. What I'm trying to say here is that the aspects of window management the user needs to handle in Windowed are the same for Window-to-Visual. So I guess I will say that

Reply:
result:
```
## Windowed hosting and Window to Visual hosting

In the Windowed hosting mode, WebView2 content is 
hosted directly in a window.  You can have multiple 
HWNDs in your app that will each be used as a WebView 
component to access web content.  The benefit of this is 
that some of the Input/Output commands are handled for 
you by the OS or by the framework. However, you will still 
need to handle some aspects of window management.

In the Window to Visual hosting mode, content is hosted 
slightly differently, but has the same benefits and 
window-management requirements as Windowed hosting.
```